src/development/packages-and-plugins/plugin-testing.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Plugins in Flutter tests
+short-title: Plugin tests
+description: Adding plugin as part of your Flutter tests.
+---
+
+Almost all [Flutter plugins][] have two parts:
+
+* Dart code, which provides the API your code calls.
+* Code written in a platform-specific (or "host") language,
+  such as Kotlin or Swift, which implements those APIs.
```

Comment:
Suggestion: Kotlin, Swift, JavaScript, or C++

Desktop and Web ftw =)

Reply:
Can web plugins be written in JavaScript? I thought they were always Dart (with calls to JS in many cases)--and referencing Dart here makes things needlessly confusing.

I didn't want to make this an exhaustive list, since the full list is quite long (since it also includes C, Objective-C, and Java, before even getting into third-party embeddings that use other languages).